oag/ja-jp/Content/Topics/Access-Gateway/add-admin-app-okta-group.htm

```diff
@@ -294,7 +294,7 @@ <h1>Okta orgグループを追加する</h1>
                                                         <p> <span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway</span>アプリへのユーザーアクセスを組織化するのにグループを使用するのはベストプラクティスです。アプリ固有のグループを作成するか、既存のグループを再利用することができます。「ユーザー、グループ、プロファイル」を参照してください。<br /></p>
                                                         <p><span class="mc-variable okta-variables.ProductName variable">Okta</span> orgのグループを定義する方法：</p>
                                                         <ol>
-                                                            <li value="1"><span class="mc-variable okta-feature-names.Administrator_dashboard variable">管理コンソール</span>で、<span class="menucascade"><span class="uicontrol">［Directory（ディレクトリ）］</span><span class="uicontrol">［Groups（グループ）］</span></span>に移動します。</li>
+                                                            <li value="1"><span class="mc-variable okta-feature-names.Administrator_dashboard variable">Admin Console</span>で、<span class="menucascade"><span class="uicontrol">［Directory（ディレクトリ）］</span><span class="uicontrol">［Groups（グループ）］</span></span>に移動します。</li>
```

Comment:
Not sure why this is changed to EN. Is `Admin Console` supposed to be untranslated?

Reply:
yes, it's DNT